scripts/cocktailsToMakeModel.js

```diff
@@ -0,0 +1,54 @@
+var cocktailsToMake = {
+  ingredientString: '',
+  paramString: '',
+  cocktailObjects: [],
+  listTemplate: '',
+
+  createIngredientString: function (){
+    $('#addedIngredients li').each(function(index){
+      cocktailsToMake.ingredientString += ('"' + $(this).text() + '"' + ', ');
```

Comment:
How do I connect the autocomplete ingredient with its ID?  The autocomplete takes only an array of names.